packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -538,6 +539,22 @@ const success = new stepfunctions.Fail(this, 'Fail', {
 });
 ```
 
+### Map
+
+A `Map` state can be used to run a set of steps for each element of an input array.
+A Map state will execute the same steps for multiple entries of an array in the state input.
+
+While the Parallel state executes multiple branches of steps using the same input, a Map state will
+execute the same steps for multiple entries of an array in the state input.
+
+```ts
+const map = new stepfunctions.Map(this, 'Map State', {
+    maxConcurrency: 1,
+    itemsPath: stepfunctions.Data.stringAt('$.inputForMap')
+});
+map.iterator(new stepfunctions.Pass(this, 'Pass State'));
```

Comment:
Sorry I missed this from last time. 

Curious: Is this a valid example - would we use Map on a Pass state? Maybe it transforms the output?

Reply:
it's valid since the iterator takes an [`IChainable`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-stepfunctions.IChainable.html). We also do this in a few of our tests for the `Map` functionality. I tested a deploy and it's valid in terms of definition